src/main/scala/no/ndla/articleapi/controller/ArticleControllerV2.scala

```diff
@@ -432,34 +432,5 @@ trait ArticleControllerV2 {
         case None           => NotFound()
       }
     }
-
-    patch(
```

Comment:
Har ikke så veldig sterke meninger fra eller til, men hva var argumentet for å flytte denne igjen?

Reply:
At vi ikke skulle bruke denne direkte, men kun via draft-api.